test/core/runtime/wasm_executor_test.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "runtime/impl/wasm_executor.hpp"
+
+#include <fstream>
+
+#include <binaryen/wasm-binary.h>
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+#include "core/extensions/mock_extension.hpp"
+#include "runtime/impl/wasm_memory_impl.hpp"
+
+using kagome::common::Buffer;
+using kagome::extensions::MockExtension;
+using kagome::runtime::WasmExecutor;
+using kagome::runtime::WasmMemoryImpl;
+
+using ::testing::Return;
+
+namespace fs = boost::filesystem;
+
+class WasmExecutorTest : public ::testing::Test {
+ public:
+  void SetUp() override {
+    extension_ = std::make_shared<MockExtension>();
+    memory_ = std::make_shared<WasmMemoryImpl>();
+
+    EXPECT_CALL(*extension_, memory()).WillRepeatedly(Return(memory_));
+    executor_ = std::make_shared<WasmExecutor>(extension_);
+  }
+
+  Buffer getSumTwoCode() {
+    // get file from wasm/ folder
+    auto path = fs::path(__FILE__).parent_path().string() + "/wasm/sumtwo.wasm";
+    std::ifstream ifd(path, std::ios::binary | std::ios::ate);
+    int size = ifd.tellg();
```

Comment:
i have a feeling that this pattern (read file with code, then execute it) will be used many times. Please, extract it to some base fixture under `test/testutils`, so others can reuse it